phantom-dsl/src/main/scala/com/websudos/phantom/util/ByteString.scala

```diff
@@ -509,16 +513,16 @@ sealed abstract class CompactByteString extends ByteString with Serializable {
  * The created ByteString is not automatically compacted.
  */
 final class ByteStringBuilder extends mutable.Builder[Byte, ByteString] {
-  builder ⇒
+  builder =>
 
-  import ByteString.{ ByteString1C, ByteString1, ByteStrings }
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://www.codacy.com/app/flavian/phantom_2/file/2220904841/issues/source?bid=2847375&fileBranchId=3173244#l518)